aws_cloudwatch_policies/aws_cloudwatch_sensitive_loggroup_encrypted.yml

```diff
@@ -0,0 +1,88 @@
+AnalysisType: policy
+Filename: aws_cloudwatch_sensitive_loggroup_encrypted.py
+PolicyID: AWS.CloudWatchLogs.SensitiveLogGroup.Encryption
+DisplayName: Sensitive AWS CloudWatch Log Encryption
+Enabled: false
+ResourceTypes:
+  - AWS.CloudWatch.LogGroup
+Tags:
+  - AWS
+  - Configuration Required
+  - Panther
+Severity: High
```

Comment:
Could go either way on `Medium` or `High`. Our three other rules about encryption at rest are `High` though.

If you get an alert, and you define exposing information as a breach....you've already got a breach. You want to contain it, though.

It seems like we currently have a fairly heavy weight of `Severity: High` rules in the entire set. May be a function of our relatively short dev history (focusing on relatively more important things first), and a broader convo than a single rule. I'm not convinced everything about encryption should be `High`, though.